test-refined/shared/src/main/scala/zio/test/refined/NumericInstances.scala

```diff
@@ -0,0 +1,63 @@
+package zio.test.refined
+
+import eu.timepit.refined.api.Refined
+import eu.timepit.refined.internal.WitnessAs
+import eu.timepit.refined.numeric.{Greater, Less}
+import zio.random.Random
+import zio.test.Gen
+
+object numeric extends NumericInstances
+
+trait NumericInstances {
+
+  def intGreaterThan[N](implicit
+                        wn: WitnessAs[N, Int]
+                       ): Gen[Random, Int Refined Greater[N]] =
+    Gen.int(wn.snd, Int.MaxValue).map(Refined.unsafeApply)
```

Comment:
I think you need to run `sbt fix` in `zio-test-refined` module

Reply:
Ran `fix` and `fmt` but no changes in the src